src/Jenssegers/Mongodb/Relations/MorphMany.php

```diff
@@ -19,4 +20,16 @@ protected function whereInMethod(EloquentModel $model, $key)
     {
         return 'whereIn';
     }
+
+    /**
+     * @inheritdoc
```

Comment:
i suggest to change or remove this doc block

Reply:
Done.